doc/release-notes/5.12-release-notes.md

```diff
@@ -0,0 +1,171 @@
+# Dataverse Software 5.12
```

Comment:
@sekmiller you should do a `git rm` of all the "snippets" in `doc/release-notes` as you incorporate them into this file.

Reply:
I was about to mention the same thing, just noticed Phil's comment. Yes, this is part of our usual procedure, to delete all the individual notes, once they have been merged into the combined release notes.

Reply:
Since this was my first time doing this I wanted to leave the other docs for reference until everyone got a chance to take a look. I will remove them now.

Reply:
Makes sense. 
Also, once they are gone, they will still be showing as "deleted" here, in the "Files Changed" tab. So it'll still be easy to double-check something in an individual note, if needed.